policyengine_canada/variables/gov/cra/tax/income/credits/school_supply_credit/school_supply_credit.py

```diff
@@ -0,0 +1,16 @@
+from policyengine_canada.model_api import *
+
+
+class school_supply_credit(Variable):
+    value_type = float
+    entity = Person
+    label = "School supply credit"
+    unit = CAD
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        expenses = person("teaching_supplies_expenses", period)
+        p = parameters(period).gov.cra.tax.income.credits.school_supply_credit
+        rate = p.amount
+        cap = p.cap
+        return min_(expenses * rate, cap)
```

Comment:
Can remove lines 14-15
```suggestion
        return min_(expenses * p.rate, p.cap)
```